apps/api/src/campaign/campaign.controller.ts

```diff
@@ -128,12 +128,9 @@ export class CampaignController {
   }
 
   @Get('byId/:id')
```

Comment:
This endpoint is for administrators only, and the returned data by it is not meant to be public. 
If the goal is to get the donation data of campaign, please use one of the available Public() endpoints